scripts/src/main/resources/scripts/command/vscode

```diff
@@ -55,7 +55,7 @@ function doAddPlugins() {
 
 function doAddPlugin() {
   doEchoAttention "Installing plugin (${1}) into VS code...\nPlease wait until VSCode has successfully installed all plugins.\nYou may then use or close VSCode as you like."
-  doRunCommand "'${VSCODE_HOME}/bin/code' --install-extension '${1}' &"
+  doRunVsCode --force --install-extension "${1}" &
```

Comment:
I like the idea of installing a software package customized and updated by the software mechanics of the ide. Anyhow, just for curiosity as I see this method. Is there an additional setup description like for eclipse which plugins to install after provisioning of vscode?
If so, how do these two mechanisms relate to each other?

Reply:
https://github.com/devonfw/ide/pull/381

Documentation for this PR is still missing. Further it is not working as expected. I need more help here.

Reply:
Also refactored project import to be aligned with this and making code homogenous: https://github.com/devonfw/ide/pull/383